lib/pages/add_bridge/show_bottom_sheet.dart

```diff
@@ -42,20 +45,29 @@ Future<bool> showBottomSheetBridge(
                         break;
                       // For other networks
                     }
-
-                    if (result != false) {
-                      completer.complete(false);
-                    }
                   },
                 );
 
-                completer.complete(true);
+                if (result == "Not Connected") {
+                  completer.complete(
+                    true,
+                  ); // returns true if is not connected
+                } else if (result == "error" || result == 'Connected') {
+                  completer.complete(
+                    false,
+                  );
```

Comment:
could this just be refactored to `completer.complete(result == "Not Connected");` ?